src/translation.jl

```diff
@@ -35,10 +35,10 @@ function translate_optim(
 ) where {T,N}
     # The final dimension of mm is assumed to represent "image number" and is not a continuous variable.
     # Hence we should not interpolate over it.
-    qfc = BSpline(Quadratic(Free(OnCell())))
+    qfc = BSpline(Linear())
```

Comment:
Interesting that this is needed. Nominally `BFGS` (and `LBFGS`) require that the derivative be continuous, which you won't get with `Linear`. This is presumably why you get that warning `Optimization failed to converge`. However, if it gets the wrong answer with Quadratic interpolation, that's obviously not good.